sdk/typescript/entrypoint/load.ts

```diff
@@ -162,6 +163,20 @@ export async function loadValue(
       // TODO(supports subfields serialization)
       return registry.buildClass(objectType, value)
     }
+    case TypeDefKind.EnumKind: {
+      const enumType = (type as TypeDef<TypeDefKind.EnumKind>).name
+
+      // Workaround to call to get the enum that has an id
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      if (dag[`load${enumType}FromID`]) {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        return dag[`load${enumType}FromID`](value)
+      }
```

Comment:
Since Enum are not loadable, I should remove this part, and potentielly just return the value, since it should have been validated by the engine first!